website/package.json

```
@@ -9,7 +9,7 @@
     "build": "next build && next-sitemap --config next-sitemap.config.cjs && next export -o dist"
   },
   "dependencies": {
-    "@theguild/components": "^5.2.2",
+    "@theguild/components": "6.0.0",
```

Please dont use exact versions.

```suggestion
    "@theguild/components": "^6.0.0",
```

---

my bad should check another websites